docs/modules/operation/pages/events/sources/kafka.adoc

```diff
@@ -0,0 +1,50 @@
+
+[[ga-events-sources-kafka]]
+
+= Kafka Consumer (Events)
+
+Events can also be created by routing them to a specific topic on Kafka.
+The Kafka Consumer feature allows events to be consumed from specific topic on Kafka.
+
+Events should be created in protobuf format link:https://developers.google.com/protocol-buffers/[Google Protocol Buffers (GPB)]
+See `kafka-consumer-events.proto`  for the model definitions.
+
+The Kafka Consumer by default consumes events from topic named `opennms-events` and forward them to `Eventd`.
+
+== Enable Kafka Consumer
+
+The Kafka Consumer is disabled by default and can be enabled as follows.
+
+First, log in to the Karaf shell of your {page-component-title} instance and configure the Kafka client settings to point to your Kafka broker.
+See link:https://kafka.apache.org/27/documentation.html#consumerconfigs[Consumer Configs] for a complete list of available options.
+
+[source]
+----
+$ ssh -p 8101 admin@localhost
```

Comment:
```suggestion
ssh -p 8101 admin@localhost
```

Reply:
We should split the source sections to indicate what is console and what is Karaf shell like this:

```
.Configure the Kafka features and the Minion location via the Karaf Shell
[source, console]
----
ssh -p 8201 admin@localhost
----

.Configure the Event consumer client
[source, karaf]
----
config:edit org.opennms.features.kafka.consumer.client
config:property-set bootstrap.servers 127.0.0.1:9092
config:update
----
```